decidim-core/lib/decidim/nicknamizable.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This concern contains the logic related to nicknames.
+  module Nicknamizable
+    def nickname_length
+      (0...20)
+    end
+
+    def nicknamize(name)
+      disambiguate(name.parameterize(separator: "_")[nickname_length])
```

Comment:
I think you can change `[nickname_length]` to `.first(nickname_length)`, and make the method like this:

```ruby
def nickname_length
  20
end
```

Reply:
I guess, hopefully we never add a minimum length and we end up changing it back! :stuck_out_tongue_closed_eyes:

Reply:
Ah, I see. It's OK then, but maybe add docs to explain why we use the range?

Reply:
Not sure after a second though. The only reason I used a range was that I was copying `devise` validatable, but a method called `nickname_length` that does not return an integer is definitely not clear enogh. I think I'm taking your suggestion after all!

Reply:
What's the status on this?